controllers/stripe.js

```diff
@@ -0,0 +1,129 @@
+const dotenv = require("dotenv");
+dotenv.config();
+const Order = require("../models/Order");
+const Stripe_Key = process.env.STRIPE_KEY;
+const stripe = require("stripe")(Stripe_Key);
+const { validationResult } = require("express-validator");
+var nodemailer = require("nodemailer");

```

Comment:
where are you using nodemailer service?

Reply:
i was using this to send email to customer